src/components/explore/QuantileExplorerView.svelte

```diff
@@ -173,9 +173,6 @@
               comparisonKeyFormatter={(perc) => `${perc}%`}
               yScaleType={probeType === 'log' ? 'scalePoint' : 'linear'}
               {yDomain}
-              )
-              ),
-              ]}
```

Comment:
@edugfilho why was this removed?

Reply:
Seems like this was leftover from another PR. None of those characters were closing anything. A local lint error on line 177 drew my attention to this.